app/controllers/internal/feedback_messages_controller.rb

```diff
@@ -2,22 +2,29 @@ class Internal::FeedbackMessagesController < Internal::ApplicationController
   layout "internal"
 
   def index
-    @q = FeedbackMessage.
-      includes(:reporter,
-               :offender,
-               :affected).
-      order("feedback_messages.created_at DESC").
+    @q = FeedbackMessage.includes(:reporter, :offender, :affected).
```

Comment:
What is `@q`? Can we rename this to provide a bit more context?

Reply:
TBH I just didn't want to go down a rabbit hole of renaming instance methods as it's already a huge PR.

`@q` is a ransack object, see https://github.com/activerecord-hackery/ransack#in-your-controller - basically a query builder and engine in one.

I'd like to keep it like that, maybe we can make a mental note to change it in a specific PR, it shouldn't be too hard to find a better name for that :D